README.rst

```diff
@@ -2,7 +2,7 @@
 PyArbTools: Keysight Signal Generator Control & Waveform Creation
 =================================================================
 
-*Current version: 2020.09.2*
+*Current version: 2021.01.15*
```

Comment:
Version should be `2021.01.1`. I have been breaking the versions down by month. So `2021.01.1` would be the first update in Jan 2021.